src/loaddb/load_db.c

```diff
@@ -186,13 +186,27 @@ ldr_validate_object_file (const char *argv0, load_args * args)
       return ER_FAILED;
     }
 
-  if (args->cs_mode && (args->load_only == true
-			|| args->index_file.empty () == false
-			|| args->schema_file.empty () == false
-			|| args->schema_file_list.empty () == false || args->trigger_file.empty () == false))
+  if (args->cs_mode)
```

Comment:
```suggestion
  if  (args->cs_mode == true)
```
the reason is also written in 193 line comment.

Reply:
I'm usually doing a comparison only if it's FALSE.